package-lock.json

```
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "thecyberhub.org",
-      "version": "0.53.12",
+      "version": "0.53.14",
```

```suggestion
      "version": "0.53.12",
```

we actually never push other files other than we have installed anything new in the packages 
can u fix this is package-lock.json

---

Yes for sure

---

> Yes for sure

if u need any help or having any doubts, let us know